.drone.yml

```diff
@@ -0,0 +1,96 @@
+branches:
+  - dev
+  - master
+  - release-*
+  - testing
+
+__buildenv: &buildenv
```

Comment:
I was aware of yaml anchors, but didn't remember the syntax, especially for dereferencing them - and was quite confused by the `__`, the `&`, and `<<: *`. Having a comment here would let me understand that faster.

Please add a comment saying `# Common parts` here, and `# Build definition` above `clone` :)

Reply:
Done.